backend/app/services/demo_posts_seed_data.rb

```
@@ -0,0 +1,39 @@
+    { nickname: '空条承太郎', body: '平静な顔で内心だけ大荒れ' },
+    { nickname: '坂田銀時', body: 'やる日は来ると言い続けて週末終わる' },
+    { nickname: '黒崎一護', body: '頼まれると結局断れずに残る' },
+    { nickname: '江戸川コナン', body: '違和感だけ先に見つけて説明できない' },
+    { nickname: '桜木花道', body: '根拠はないのに自信だけは満点' },
+    { nickname: 'ケンシロウ', body: '静かな怒りが顔にだけ出る' },
+    { nickname: '越前リョーマ', body: 'まだ本気出してない感だけうまい' },
+    { nickname: '緋村剣心', body: '丸く収めたあとで一人反省会する' },
+    { nickname: 'サイタマ', body: '全力出す前に用事が終わってる' },
+    { nickname: '両津勘吉', body: 'ひらめきは天才、詰めで全部こぼす' },
+    { nickname: '月野うさぎ', body: '朝だけ感情の起伏が魔法少女級' },
+    { nickname: '木之本桜', body: '片付けたはずの机が夕方また散らかる' }
```

RuboCop (1.85.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)